content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -177,6 +197,45 @@ spec:
         cpu: "500m"
 ```
 
+## Pod resources example {#example-2}
+
+<!-- feature gate PodLevelResources -->
+{{< feature-state for_k8s_version="v1.32" state="alpha" >}}
+
+This following Pod runs two containers, log-aggregator and app. The Pod has an
+explicit request of 1 CPU and 128 MiB of memory, and an explicit limit of 2 CPU and
+256 MiB of memory. The log-aggregator container has explicit requests and limits
+set. However, the app container will simply share the resources available within the
+Pod resource boundaries, as it does not have explicit requests and limits set.
+
+```yaml
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: frontend
+spec:
+  resources:
+    requests:
+      memory: "128Mi"
+      cpu: "1"
+    limits:
+      memory: "256Mi"
+      cpu: "2"
+  containers:
+  - name: app
+    image: images.my-company.example/app:v4
+  - name: log-aggregator
+    image: images.my-company.example/log-aggregator:v6
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "250m"
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
```

Comment:
We recommend using `code_example` shortcodes for entire example manifests; see https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files (which you are using elsewhere).

Reply:
yeah for this file, the other example is not using code_example shortcodes, so for consistency I kept it same. Let me fix it